airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/config.py

```diff
@@ -143,6 +143,12 @@ class TestConfig(BaseConfig):
 
 
 class Config(BaseConfig):
+    class StrictMode(str, Enum):
+        strict = "strict"
```

Comment:
Language nuance:
* If the only option is strict, are we maybe talking about a `strict` boolean?
* If we are talking about multiple options (e.g. `strict`, `medium`, `light`) then we aren't talking about `strict_mode`, we have a `test_mode` perhaps?

Reply:
Yes it's fair! I renamed to `test_mode`